base/log_keys.go

```diff
@@ -59,6 +60,7 @@ var (
 		KeyHTTP:           "HTTP",
 		KeyHTTPResp:       "HTTP+", // Infof printed as HTTP+
 		KeyImport:         "Import",
+		KeyJavascriptLog:  "JavascriptLog",
```

Comment:
Feel free to push back, but I think it's already implied it's a log. We could shorten the string to `"JS"`, or just keep `"JavaScript"`?

Reply:
Nope, thats a completely fair suggestion :)

Reply:
I'm going to change to Javascript to be explicit